renderer-macos/lib/stubs/NSTextView_stubs.c

```diff
@@ -11,6 +11,10 @@
   if (self) {
     self.attributedString = [[NSMutableAttributedString alloc] init];
     self.attributedProps = [NSMutableDictionary dictionary];
+    self.paragraphStyle =
```

Comment:
I'd rename this file to `BRISKTextView` (as well as the class). I think it'll serve as good organisational method. What's more I think that `.h` files should be kept as empty as possible in the spirit of not creating rules that are not verified automatically. We can set an easy to follow convention. Like, stubs used by OCaml should always be prefixed with something.

Reply:
and actually I'd leave anything that can be Objective-C in Objective-C for now. It's a little bit arbitrary but it'll keep the code contained in the objc class. If we ever need to migrate/share this logic, we'll reconsider it.